ts/defs.bzl

```diff
@@ -378,6 +378,7 @@ def ts_project(
             # that this js_library can be a valid dep for downstream ts_project or other rules_js derivative rules.
             srcs = [transpile_target_name, tsc_target_name],
             deps = deps,
+            data = data,
```

Comment:
Should `assets` also go into the `data`?

Reply:
assets should go in `srcs` actually

Reply:
can you put that up in a separate PR?